eisbuk-admin/src/components/atoms/AttendanceCard/__tests__/AttendanceCard.test.tsx

```diff
@@ -18,21 +19,27 @@ const mockDispatch = jest.fn();
 jest.mock("react-redux", () => ({
   useDispatch: () => mockDispatch,
 }));
+
+const spyT = jest.spyOn(i18n, "t");
+beforeEach(() => spyT.mockClear());
 describe("AttendanceCard", () => {
   describe("Smoke test", () => {
     beforeEach(() => {
-      render(<AttendanceCard {...dummySlot} />);
+      render(<AttendanceCard {...customersSlot} />);
+      () => spyT.mockClear();
     });
     test("should render props", () => {
       screen.getByText("13:00 - 15:00");
       screen.getByText("Saul");
       screen.getByText("Walter");
-      screen.getByText(Category.Competitive);
+      // screen.getByText(Category.Competitive);
+      expect(spyT).toHaveBeenCalledWith(`Categories.${Category.Competitive}`);
+      expect(spyT).toHaveBeenCalledWith(`SlotTypes.${SlotType.Ice}`);
```

Comment:
Nice...so yeah, we're implementing this for now but will return to `screen.getBytext` when we implement `i18next` with tests.

Reply:
So, yeah...maybe just leave a `@TODO` here.